### PR TITLE
Percentages of citizen datapoints mapping to NA reference values

### PR DESCRIPTION
**Pull request by jacobe90**
- data cleaning changes from [https://github.com/BU-Spark/pitne-season-watch/pull/7](https://github.com/BU-Spark/pitne-season-watch/pull/7)
- made table of percentage of observations mapping to NA values in ref data, see `reference and citizen data validation.ipynb`

### CODE REVIEW

File: Untitled.ipynb
Comment (zacharymeurer) [inline Untitled.ipynb:1]: Delete.

**Comment by jacobe90** [comment]
Not quite ready for merging yet, need `user_tree_id`, `lat`, and `lon` to be added back to the citizen datapoints

File: Untitled.ipynb
Comment (jacobe90) [reply Untitled.ipynb:1]: done

**Review by AnaSof0 — APPROVED** [review]

File: pvt_cleaning.ipynb
Comment (zacharymeurer) [inline pvt_cleaning.ipynb:1]: 'NA' should not map to -2 values in the reference database. They should probably just map to None or NaN values instead. This same logic likely applies to mapping '' to -1.

**Review by zacharymeurer — APPROVED** [review]
lgtm

File: pvt_cleaning.ipynb
Comment (colettebas) [inline pvt_cleaning.ipynb:1]: Please make sure all TODOs (here and in the other notebooks) are captured as tasks in Notion!

File: data cleaning.ipynb
Comment (colettebas) [inline data cleaning.ipynb:1]: I noticed that
`bihar_df = pd.read_csv("citizenData/Bihar_citizenData.csv")`
is causing a FileNotFound. Can you verify that this data is being used if it is needed?

**Review by colettebas — COMMENTED** [review]
Two small notes but it otherwise looks good!

File: pvt_cleaning.ipynb
Comment (jacobe90) [reply pvt_cleaning.ipynb:1]: Added todo to check with geetha that nothing in reference data should map to -1 in the next meeting.

File: data cleaning.ipynb
Comment (jacobe90) [reply data cleaning.ipynb:1]: Removed from this cell - was example code.

**Pull request merged**